docs/concepts.rst

```diff
@@ -1,128 +1,130 @@
 
-Learn more about managing unpackaged metadata in TODO: link Managing unpackaged configuration.
+Learn more about managing unpackaged metadata in [TODO: link Managing unpackaged configuration].
 
 Project Orgs & Services
 -----------------------
 
-Orgs and services are external, authenticated resources that each project uses. CumulusCI makes it easy to connect orgs, as well as services like a GitHub account or a MetaDeploy instance, to a single project or to use them across projects. 
+Orgs and services are external, authenticated resources that each project uses. CumulusCI makes it easy to connect orgs and services to a single project, or to use them across many projects.
 
 Orgs
 ^^^^
 
-Each project has its own set of orgs, including active scratch orgs, persistent orgs like a production or packaging org, and predefined scratch org configurations. CumulusCI securely stores org authentication information in its keychain, making it easy to access connected orgs at any time. The ``cci org list`` command shows all of the orgs connected to a project. Orgs can also be `shared across multiple projects <TODO: link to persistent orgs section>`_.
+Each project has its own set of orgs, including active scratch orgs, persistent orgs like a production or packaging org, and predefined scratch org configurations. CumulusCI securely stores org authentication information in its keychain (via a unique identifier of characters), making it easy to access connected orgs at any time. The ``cci org list`` command shows all of the orgs connected to a project. Orgs can also be shared across multiple projects.
 
-Configuring orgs in CumulusCI is powerful, but comes with some complexity. To review all of the details, read the sections TODO: link "Scratch org environments" and TODO: link persistent org section.
+Configuring orgs in CumulusCI is powerful, but comes with some complexity. To review the details, read the sections [TODO: link Scratch org environments] and [TODO: link persistent org section].
 
 Services
 ^^^^^^^^
 
-Services represent external resources used by CumulusCI automation, such as access to a GitHub account or a MetaDeploy instance. Services are usually, but not always, connected to CumulusCI across projects as part of the global keychain. 
+Services represent external resources used by CumulusCI automation, such as access to a GitHub account or a MetaDeploy instance. Services are usually, but not always, connected to CumulusCI across projects as part of the global keychain. The command ``cci service list`` shows you which services are connected in the context of the current project.
 
-The command ``cci service list`` shows you which services are connected in the context of the current project.
+Global services are easy to use and share and, therefore, we recommend you rely on them for the most part. However, services can also be connected at the project level, which means that they're scoped to a single project and cannot be shared.
```

Comment:
```suggestion
Global services are easy to use and share, and we recommend that you use them when possible. However, services can also be connected at the project level, which means that they're scoped to a single project and can't be shared.
```

Reply:
"Global services are easy to use and share; we recommend that you use them as much as possible."

More conversational in tone. And I'm still gonna advocate for the "cannot" here for emphasis.

Reply:
Sounds good! That semicolon seems fussy for such a short sentence. Replace with a period?

Roger on "cannot."